### PR TITLE
refactor: steps to setup project before to run it

### PR DESCRIPTION
**Pull request by eapg**
* Refactor steps to setup the project before to run in for the first time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ Download and install the following tools:
 * [checkstyle](https://maven.apache.org/plugins/maven-checkstyle-plugin/usage.html)
 * [Google java format](https://github.com/google/google-java-format)
 
+* In order to setup this project it's necessary:
```
Comment (reynoldmorel) [inline README.md:16]: You need to make sure the migrations ran in python side, so you can point to the section in the python repo that runs the migrations

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eapg) [reply README.md:16]: Done.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,21 @@ Download and install the following tools:
...
+
+  * set BD connexion parameters in `application.properties` (URL, Username, Password)
```
Comment (reynoldmorel) [inline README.md:18]: Start with caps leters

File: README.md [hunk 3]
```diff
@@ -13,6 +13,21 @@ Download and install the following tools:
...
+  * set BD connexion parameters for testing in route test/resources `applicatin.properties` (URL, Username, Password)
+  * Set oauth2.secret.key in `application.properties` as part of Oauth2 security.
+  * It is important to validate that migrations ran in python side of the project to have BD and tables available.
```
Comment (reynoldmorel) [inline README.md:21]: I think you should move this to the beginning. This the first thing you need to make sure before even starting the setup process

File: README.md [hunk 4]
```diff
@@ -13,6 +13,21 @@ Download and install the following tools:
...
+    To go deeper into the migrations part refer to: [migrations](https://github.com/eapg/EA_RESTAURANT#migrations)
+  * Run maven command `mvn package` in order to set all dependencies in `POM` file.
+  * As this is a project that implement Grpc it is necessary to run command `mvn compile` to build `.java` from
+    `.proto` files.
+  * Before to run the `springBootApp` to start the application it is necessary to set intellij to get the specific profile using
```
Comment (reynoldmorel) [inline README.md:26]: You meant `Before running`?

File: README.md [hunk 5]
```diff
@@ -13,6 +13,21 @@ Download and install the following tools:
...
 
+* In order to setup this project it's necessary:
+
+  * set BD connexion parameters in `application.properties` (URL, Username, Password)
+  * set BD connexion parameters for testing in route test/resources `applicatin.properties` (URL, Username, Password)
+  * Set oauth2.secret.key in `application.properties` as part of Oauth2 security.
+  * It is important to validate that migrations ran in python side of the project to have BD and tables available.
+    To go deeper into the migrations part refer to: [migrations](https://github.com/eapg/EA_RESTAURANT#migrations)
+  * Run maven command `mvn package` in order to set all dependencies in `POM` file.
+  * As this is a project that implement Grpc it is necessary to run command `mvn compile` to build `.java` from
```
Comment (reynoldmorel) [inline README.md:24]: You meant `Since this is`

Comment (reynoldmorel) [inline README.md:24]: `that implements`

File: README.md [hunk 6]
```diff
@@ -13,6 +13,21 @@ Download and install the following tools:
 * [checkstyle](https://maven.apache.org/plugins/maven-checkstyle-plugin/usage.html)
 * [Google java format](https://github.com/google/google-java-format)
 
+* In order to setup this project it's necessary:
```
Comment (reynoldmorel) [inline README.md:16]: `it's necessary to:`

Also make sure each point makes sense, they all should start with a verb

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (eapg) [reply README.md:18]: Done.

File: README.md (hunk 3 above)
Comment (eapg) [reply README.md:21]: Done.

File: README.md (hunk 5 above)
Comment (eapg) [reply README.md:24]: Done.

File: README.md (hunk 6 above)
Comment (eapg) [reply README.md:16]: Done.

File: README.md (hunk 4 above)
Comment (eapg) [reply README.md:26]: Done.

**Review by reynoldmorel — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -13,6 +13,23 @@ Download and install the following tools:
...
 
+* In order to setup this project it's necessary to:
+
+  * Validate that migrations ran in python side of the project to have BD and tables available.
+    To go deeper into the migrations part refer to: [migrations](https://github.com/eapg/EA_RESTAURANT#migrations)
+  * Set BD connexion parameters in `application.properties` (URL, Username, Password).
+  * Set BD connexion parameters for testing in route test/resources `applicatin.properties` (URL, Username, Password).
+  * Set oauth2.secret.key in `application.properties` as part of Oauth2 security.
+  * Run maven command `mvn package` in order to set all dependencies in `POM` file.
+  * Since this is a project that implements Grpc it is necessary to run command `mvn compile` to build `.java` from
```
[the remark this answers is not in the document]
Comment (eapg) [reply README.md]: Done.

**Pull request merged**